examples/plugin/README.md

```diff
@@ -0,0 +1,34 @@
+# Example plugin
+
+## Building
+
+From the root of the irmin repository:
+
+```shell
+$ dune build ./examples/plugin/plugin.cmxs
+```
+
+## Usage
+
+To load this plugin you can use the `--plugin` flag when using `irmin` (or
+`dune exec ./src/irmin-unix/bin/main.exe` from the root of the irmin repo):
+
+```shell
+$ dune exec ./src/irmin-unix/bin/main.exe -- set --plugin _build/default/examples/plugin/plugin.cmxs a/b/c 123
+```
```

Comment:
This works, but if I try `get` after, while using the plugin, I get `<none>`. Is this expected? 
Related question: since there are no tests for the plugins yet, do you think its possible to run mdx on this readme? (I'm not sure what are the limitations for mdx).

Reply:
It's kind of a weird example because the plugin adds an in-memory store that uses integers, so `get` returning nothing is to be expected since it's opening a new in-memory store each time.

I will try running mdx for this, that's a good idea!

Reply:
Just to understand, is this because we cannot use an in-memory store from irmin-unix in general or is it specific to how plugins work?

Reply:
For commands like get/set/remove/… an in-memory store will create a new instance each time, so those won’t work as expected. An in-memory store is more useful with the graphql or http server when it can be accessed from another process.

Reply:
I see, thanks for the explanations!